src/Schema/Schema.php

```diff
@@ -956,4 +955,66 @@ public function getSchemaCache(): SchemaCache
     {
         return $this->schemaCache;
     }
+
+    /**
+     * This method will call a `'loadTable' . ucfirst($type)` named method with the table name to obtain the metadata.
+     *
+     * @param string $type
+     * @param string $name
+     *
+     * @return mixed
+     */
+    protected function loadTableType(string $type, string $name)
+    {
+        switch ($type) {
+            case SchemaInterface::SCHEMA:
+                return $this->loadTableSchema($name);
+            case SchemaInterface::PRIMARY_KEY:
+                return $this->loadTablePrimaryKey($name);
```

Comment:
Maybe in this PR add these and other methods to `SchemaInterface` ?

Reply:
All methods added to SchemaInterface